examples/react/playground/src/stories/UploadsList.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { CID } from 'multiformats/cid'
+import * as Link from 'multiformats/link'
```

Comment:
Note: CID still exists but `Link` is designed to be more abstract and less prone to becoming out of date.